### ISSUE TITLE
Exporting a global, then using "import * as", then calling a method on that imported global, loses the method property access

### ISSUE THREAD

**Issue by Conduitry** [rollup/rollup#570]
This issue appeared in 0.25.5.

main.js:
```javascript
import * as moduleContents from './global-exporter';
moduleContents.exportedGlobal.someMethod();
```

global-exporter.js:
```javascript
export var exportedGlobal = someGlobal;
```

result:
```javascript
'use strict';

var exportedGlobal = someGlobal;

exportedGlobal();
```

the `.someMethod` gets eaten, and the resulting code tries to call the export directly as a function.

Playground link:

http://rollupjs.org/#{%22options%22%3A{%22format%22%3A%22cjs%22%2C%22moduleName%22%3A%22myBundle%22%2C%22globals%22%3A{}}%2C%22modules%22%3A[{%22name%22%3A%22main.js%22%2C%22code%22%3A%22import%20*%20as%20moduleContents%20from%20%27.%2Fglobal-exporter%27%3B\nmoduleContents.exportedGlobal.someMethod%28%29%3B%22}%2C{%22name%22%3A%22global-exporter.js%22%2C%22code%22%3A%22export%20var%20exportedGlobal%20%3D%20someGlobal%3B%22}]}

**Comment by Conduitry** [comment]
Actually, fiddling a bit more, it doesn't seem to be specific to exporting a global:

http://rollupjs.org/#{%22options%22%3A{%22format%22%3A%22cjs%22%2C%22moduleName%22%3A%22myBundle%22%2C%22globals%22%3A{}}%2C%22modules%22%3A[{%22name%22%3A%22main.js%22%2C%22code%22%3A%22import%20*%20as%20moduleContents%20from%20%27.%2Fmodule%27%3B\nmoduleContents.exportedObject.someMethod%28%29%3B%22}%2C{%22name%22%3A%22module.js%22%2C%22code%22%3A%22export%20var%20exportedObject%20%3D%20{%20someMethod%3A%20function%20%28%29%20{%20}%20}%3B%22}]}

Accessing the property for the method call still is lost.

**Comment by Victorystick** [comment]
@Conduitry This is most certainly due to a regression in #566. A fix for it is inbound as of #569.

**Comment by Rich-Harris** [comment]
Looks like this is fixed and released! Closing
